code/src/components/GlobalStyles.js

```diff
@@ -0,0 +1,39 @@
+import styled, { createGlobalStyle } from 'styled-components';
+
+export const GlobalStyle = createGlobalStyle`
+
+`
+
+export const OuterWrapper = styled.div`
+  background-color: #faf6f0;
+;
+`
+
+export const InnerWrapper = styled.section`
+max-width: 940px;
+width: 90%;
+height: 100vh;
```

Comment:
Set height to 100% if you want the coral color to stretch the entire page. Otherwise, nvm